GVRf/Framework/framework/src/main/jni/engine/renderer/gl_renderer.cpp

```diff
@@ -282,8 +284,8 @@ namespace gvr
                 renderPostEffectData(rstate, input_texture, post_effects, i);
                 input_texture = renderTexture;
             }
-            GL(glBindFramebuffer(GL_FRAMEBUFFER, saveRenderTexture->getFrameBufferId()));
-            GL(glViewport(0, 0, saveRenderTexture->width(), saveRenderTexture->height()));
+            GL(glBindFramebuffer(GL_FRAMEBUFFER, drawFboId));
+            GL(glViewport(0, 0, viewport[2], viewport[3]));
```

Comment:
If we are restoring a viewport, how about restoring all 4 values? Don't assume old viewport always starts at (0,0)